Header header
float32 rightEyeYaw
float32 rightEyePitch
float32 leftEyeYaw
float32 leftEyePitch